app/react/Metadata/components/specs/__snapshots__/ValueList.spec.js.snap

```diff
@@ -47,27 +45,19 @@ exports[`ValueList should render the values as a ul list 1`] = `
   className="multiline"
 >
   <li
-    key="labelfirst_value"
+    key="NaN"
```

Comment:
All this NaNs are also probably a indicators that fixtures require changing, or at least reviewing.  These had values on the previous version.